specification/iotcentral/data-plane/Microsoft.IoTCentral/stable/1.0.0/iotcentral.json

```diff
@@ -0,0 +1,2262 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DeviceProperties"
+            }
+          }
+        }
+      },
+      "put": {
+        "operationId": "Devices_ReplaceProperties",
+        "summary": "Replace device properties",
+        "description": "Replace all property values of a device by device ID.",
+        "x-ms-examples": {
+          "Update properties": {
+            "$ref": "./examples/devices_properties_update.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/DeviceId"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Device properties.",
+            "schema": {
+              "$ref": "#/definitions/DeviceProperties"
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DeviceProperties"
+            }
+          }
+        }
+      },
+      "patch": {
+        "operationId": "Devices_UpdateProperties",
+        "summary": "Update device properties via patch",
+        "description": "Update property values of a device by device ID.",
+        "x-ms-examples": {
+          "Update properties": {
+            "$ref": "./examples/devices_properties_update.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/DeviceId"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Device properties patch.",
+            "schema": {
+              "type": "object"
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DeviceProperties"
+            }
+          }
+        }
+      }
+    },
+    "/devices/{device_id}/telemetry/{telemetry_name}": {
+      "get": {
+        "operationId": "Devices_GetTelemetryValue",
+        "summary": "Get device telemetry value",
+        "description": "Get the last telemetry value from a device.",
+        "x-ms-examples": {
+          "Get telemetry": {
+            "$ref": "./examples/devices_telemetry_get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/DeviceId"
+          },
+          {
+            "$ref": "#/parameters/TelemetryName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DeviceTelemetry"
+            }
+          }
+        }
+      }
+    },
+    "/roles": {
+      "get": {
+        "operationId": "Roles_List",
```

Comment:
Does this and other list calls work with a continuation token?

Reply:
By using this:
```
        "x-ms-pageable": {
          "nextLinkName": "nextLink"
        },
```

in the Swagger, this means the service must return a JSON that contains an absolute URL that we can GET as the `nextLink` key in the payload, and also this implies the is an array type objects in the `value` node. If that is correct, then this operation will be correctly pageable in SDK